mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule JemallocStats.MixProject do
   def project do
     [
       app: :jemalloc_info,
-      version: "0.4.0",
+      version: "0.7.0",
```

Comment:
it felt very weird/potentially misleading to just bump this to 0.5, if i was bumping the rust side to 0.7